src/test/java/com/epam/esm/security/UserSecurityUtilTest.java

```diff
@@ -0,0 +1,95 @@
+package com.epam.esm.security;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.security.authentication.AuthenticationServiceException;
+import org.springframework.security.core.userdetails.UserDetailsService;
+
+@SpringBootTest
```

Comment:
Why do you need this annotation for unit tests?

Reply:
To properly mock objects